gliner/model.py

```diff
@@ -640,7 +641,7 @@ def prepare_state_dict(self, state_dict):
         """
         new_state_dict = {}
         for key, tensor in state_dict.items():
-            key = re.sub("_orig_mod\.", "", key)
+            key = re.sub(r"_orig_mod\.", "", key)
```

Comment:
Added regex matching for valid expression of dot
https://stackoverflow.com/questions/13989640/regular-expression-to-match-a-dot